### PR TITLE
[IMP] New command line option to generate default nginx and odoo conf…

### PR DESCRIPTION
**Pull request by nseinlet**
Description of the issue/feature this PR addresses:
[IMP] New command line option to generate default nginx and odoo config file
Using workers, proxy_mode and ssl by default

Current behavior before PR:
Starting point for configuration is google

Desired behavior after PR is merged:
A command line to generate a generic config file, based on current hardware
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

@KangOl

### CODE REVIEW

File: openerp/cli/conf/odoo.conf.template [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[options]
+logfile = /var/log/odoo/{{hostname|snake}}.log
+logrotate = True
```
Comment (KangOl) [inline openerp/cli/conf/odoo.conf.template]: Wont work well with workers (wwwww rules)

**Comment by KangOl** [comment]
nice addition: use `tools.find_in_path('nginx')` and warn if not installed.
Same for a `logrotate` config

File: openerp/cli/tune.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        tune_template('.').render_to(
+            '.',
+            directory(args.dest, create=True),
+            opts)
+            
+        for thetool in ('nginx', 'logrotate'):
+            try:
+                tools.find_in_path(thetool)
+            except IOError:
+                print 'Warning : Unable to find %r in path' % (thetool,)
```
Comment (KangOl) [inline openerp/cli/tune.py]: use `warn` function

File: openerp/cli/tune.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        self.path = identifier
+        if os.path.isdir(self.path):
+            return
+        die("{} is not a valid module template".format(identifier))
+        
+def die(message, code=1):
+    print >>sys.stderr, message
+    sys.exit(code)
+
+def warn(message):
```
Comment (KangOl) [inline openerp/cli/tune.py]: unused

**Comment by KangOl** [comment]
Also, in order to make runbot happy about the CLA, you should change your commit email for this repository.

**Comment by dreispt** [comment]
Missing update to the docs 😿

**Comment by bealdav** [comment]
:+1:

**Comment by legalsylvain** [comment]
Could you explain https exception for POS ? (--no-pos / --pos option).
I currently using odoo on https with point of sale. 
Otherwise, nice features, and good idea ! Thanks.

**Comment by nseinlet** [comment]
Maybe this option should better be called --no-posbox / --posbox .
This configuration is a workaround for the "Mixed Content" Security Policy
https://github.com/odoo/odoo/issues/3156

**Comment by nhomar** [comment]
Will this be merged?

**Comment by bealdav** [comment]
interesting feature

**Pull request closed without merging**